src/PAModel/Schemas/PropertyDataType.cs

```diff
@@ -7,25 +7,25 @@ namespace Microsoft.PowerPlatform.Formulas.Tools.Schemas
     // plus the addition of Screen, which is available in the component property type picker.
     internal enum PropertyDataType
     {
-        Invalid,
-        _Min,
-        Record,
-        Table,
-        Boolean,
-        Number,
-        String,
-        Date,
-        Time,
-        DateTime,
-        DateTimeNoTimeZone,
-        Hyperlink,
-        Currency,
-        Image,
-        Color,
-        Enum,
-        Media,
-        Guid,
-        Screen,
-        _Lim
+    Invalid,
```

Comment:
Indenting seems correct here. Shouldn't anything within curly braces be indented?

Reply:
I was wondering the same, but <Rule Id="SA1137" Action="None" /> <!-- ElementsShouldHaveTheSameIndentation -->
Was throwing this error before.